.github/issue-labler.yml

```
@@ -0,0 +1,3 @@
+Status: 
+  Triage:
```

Do the docs say you can do this to create a nested label name?
Remember, the label is not just `Triage`.

---

> Do the docs say you can do this to create a nested label name? Remember, the label is not just `Triage`.

what is the alternative then ?

---

> nested label name

I'm not sure what this means, but the label, per the issue, should be "Status: Triage".  You'll need to use quotes because `:` is a special char in YAML.

```suggestion
'Status: Triage':
```